### DISCUSSION TITLE
Changere/remove window titlebars

### DISCUSSION THREAD

**Discussion by gsstark** [i3/i3#4799] [Q&A] [answered]
Being a bit oldschool I find the titlebars pretty superfluous decoration. How can I turn them off or set them to be 0 height or something?

**Comment by Airblader** [comment, marked as answer]
https://i3wm.org/docs/userguide.html#default_border:

```
default_border none
default_floating_border none
```

Note that this doesn't affect titlebars for stacked or tabbed containers. Those cannot be disabled, but you can set a font of size 0 to make them smaller if you want.

**Reply by elebow** [reply]
>Note that this doesn't affect titlebars for stacked or tabbed containers. Those cannot be disabled,

Is this a design decision, or would the project consider a PR to allow this behavior?

My use case is that I have a separate task bar (tint2) that indicates the current window, so the i3 titlebar is redundant.

**Reply by okraits** [reply]
I guess the reason is that you need to have some indication that the specific container includes more than one window.
